elixir/apps/domain/lib/domain/accounts/account/changeset.ex

```diff
@@ -54,6 +54,10 @@ defmodule Domain.Accounts.Account.Changeset do
     |> cast_embed(:features, with: &Features.Changeset.changeset/2)
     |> cast_embed(:limits, with: &Limits.Changeset.changeset/2)
     |> cast_embed(:metadata, with: &metadata_changeset/2)
+    |> check_constraint(:config,
+      name: :dns_mutual_exclusivity,
+      message: "cannot have both Do53 resolvers and DoH provider configured"
```

Comment:
This message could be shown to the admin / user.

```suggestion
      message: "cannot have both custom DNS resolvers and DoH provider configured"
```

Reply:
What is wrong with the Do53 terminology? It may be a bit technical but also communicates to admins that we won't use DoH or DoT with their own resolvers (even if they specify 1.1.1.1 here for example).

Reply:
This will go away with the new data structure.